MatrixSDK/VoIP/MXCallManager.m

```diff
@@ -882,36 +882,37 @@ - (void)transferCall:(MXCall *)callWithTransferee
                         //  generate a new call id
                         NSString *newCallId = [[NSUUID UUID] UUIDString];
                         
-                        dispatch_group_t dispatchGroupReplaces = dispatch_group_create();
+                        dispatch_group_t dispatchGroup = dispatch_group_create();
+                        dispatch_group_enter(dispatchGroup);
                         
                         if (callWithTarget)
                         {
                             //  send replaces event to target
-                            dispatch_group_enter(dispatchGroupReplaces);
                             [callWithTarget transferToRoom:nil
                                                       user:transferee
                                                 createCall:nil
                                                  awaitCall:newCallId
                                                    success:^(NSString * _Nonnull eventId) {
-                                dispatch_group_leave(dispatchGroupReplaces);
+                                dispatch_group_leave(dispatchGroup);
```

Comment:
I think you also need to leave the dispatch group in case of `failure`

Reply:
> Does it create an issue (like a retain cycle) if we let the dispatch_group open

This is what I meant. 
A dispatch group with a work count equal to "one" will stick somewhere forever I guess.
But I can't find easily a confirmation of this (DispatchGroup objects don't work as normal ARC objects).
Btw I don't see it as a huge problem, I'm going to approve it anyway.